packages/os/ebs-volumes.rules

```diff
@@ -1,8 +1,10 @@
 ACTION=="remove", GOTO="ebs_volumes_end"
-SUBSYSTEM!="block", GOTO="ebs_volumes_end"
-ENV{DEVTYPE}!="disk", GOTO="ebs_volumes_end"
+
+# Create symlinks for the device name configured in EC2, for both block devices and partitions
+KERNEL=="nvme[0-9]*n[0-9]*", ENV{DEVTYPE}=="disk", ATTRS{model}=="Amazon Elastic Block Store", PROGRAM="/usr/bin/ghostdog ebs-device-name /dev/$kernel", SYMLINK+="$result"
+KERNEL=="nvme[0-9]*n[0-9]*p[0-9]*", ENV{DEVTYPE}=="partition", ATTRS{model}=="Amazon Elastic Block Store", PROGRAM="/usr/bin/ghostdog ebs-device-name /dev/$kernel", SYMLINK+="$result$number"
```

Comment:
If you set `ENV{XVD_DEVICE_NAME}` in the parent, you should be able to import it here via `IMPORT{parent}="XVD_DEVICE_NAME"` to avoid the second binary invocation.